### PR TITLE
Release 2.3.0

### PR DESCRIPTION
**Pull request by bmc-msft**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,16 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## 2.3.0
+### Changed
+* Service: Clarify log messages when the service & agent versions mismatch. [#510](https://github.com/microsoft/onefuzz/pull/510)
```
Comment (chkeita) [inline CHANGELOG.md:9]: and ?

**Review by chkeita — APPROVED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (bmc-msft) [inline CHANGELOG.md:9]:
```suggestion
* Service: Clarify log messages when the service and agent versions mismatch. [#510](https://github.com/microsoft/onefuzz/pull/510)
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -4,6 +4,16 @@ All notable changes to this project will be documented in this file.
...
 
+## 2.3.0
+### Changed
+* Service: Clarify log messages when the service and agent versions mismatch. [#510](https://github.com/microsoft/onefuzz/pull/510)
+* Service: Scalesets and Nodes are now updated in a consistent order during scheduled updates. [#512](https://github.com/microsoft/onefuzz/pull/512)
+* CLI/Service: Expanded the use of Primitive data types that provide data validation. [#514](https://github.com/microsoft/onefuzz/pull/514)
+
+### Fixed
+* Service: Fixed an error generated when scalesets scheduled for deletion had configurations updated.  [#511](https://github.com/microsoft/onefuzz/pull/511)
+* Service: Fixed an issue where scaleset configurations were updated too frequently.  [#511](https://github.com/microsoft/onefuzz/pull/511)
```
Comment (ranweiler) [inline CHANGELOG.md:15]: Intentional link duplicate?

Comment (bmc-msft) [reply CHANGELOG.md:15]: Yes.  its really two separate issues that got fixed in one PR.

**Pull request merged**